style.css

```diff
@@ -122,8 +123,8 @@ body {
   flex-direction: column;
   margin-left: auto;
   margin-right: auto;
-  margin-bottom: 100px;
-  margin-top: 0;
+  margin-bottom: 200px;
```

Comment:
### Optional
- [ ] It would be better to change the`.work-card { margin-bottom: 100px}` on mobile devices so that we can achieve the spacing between the contents as in the Figma design